app/scripts/directives/fromFile.js

```diff
@@ -118,6 +118,7 @@ angular.module("openshiftConsole")
             return;
           }
 
+          $scope.resource = resource;
```

Comment:
Resource is scoped to only this function.  How do I get it out of this function?